### ISSUE TITLE
Pixel 6a: Bluetooth Tethering switches itself off again immediately after switching it on

### ISSUE THREAD

**Issue by Professor-Berni** [GrapheneOS/os-issue-tracker#2295]
My car needs Bluetooth Tethering, but every time I'm switching it on it switches itself off again. Directly after tapping the back button Bluetooth Tethering is switched off again.
The very weird thing is, after switching it on for one time, my car can connect to the Pixel 6a until a reboot of the Pixel 6a.
It that a bug or a feature?
![Screenshot_20230704-101506](https://github.com/GrapheneOS/os-issue-tracker/assets/8540043/0936ae86-545d-441e-936d-6c4aa5d5d0c9)
![Screenshot_20230704-101515](https://github.com/GrapheneOS/os-issue-tracker/assets/8540043/939d9192-d77f-4d24-9039-057db16e38d7)

**Comment by thestinger** [comment]
This is not described in enough detail to be actionable. Have confirmed Bluetooth Tethering works as expected on the Pixel 6a. You may have an app interfering with this. You'll need to investigate that.

**Comment by Professor-Berni** [comment]
Please excuse my bad English; I'm German and English is not my mother language.

Does the switch for Bluetooth Tethering (as shown in my screenshots) remains switched on at your device after closing the setup app? Does the switch remains switched on at your device after rebooting your device?
If not, it is a bug! That would be the same as if the wifi switch (or any other switch) disables itself after reboot or closing the setup app.
In a German Android forum this issue is well diskussed: [Klick](https://www.android-hilfe.de/forum/google-pixel-6a.4073/bluetooth-tethering-schaltet-sich-immer-wieder-aus.1061468.html)

What more description do you need?

It would be very nice if an issue would be handled more seriously.

**Comment by thestinger** [comment]
So to clarify, it works fine for you until reboot, but the switch appears off even though it's working when you open that Settings app menu again? So the issue is the Settings app displays the wrong thing.

**Comment by Professor-Berni** [comment]
Yes, once activated, Bluetooth Tethering works fine until a reboot of the Pixel 6a.

I think, the Settings app doesn't store this setting. And after a reboot Bluetooth Tethering is not enabled anymore, because the Settings app restores the wrong value (or the default value [I'm guessing, OFF is the default value]) of this setting.

I hope, I can explain my thoughts in a right way...
